ios/PepperoniAppTemplate.xcodeproj/project.pbxproj

```diff
@@ -22,9 +22,7 @@
 		13B07FC11A68108700A75B9A /* main.m in Sources */ = {isa = PBXBuildFile; fileRef = 13B07FB71A68108700A75B9A /* main.m */; };
 		140ED2AC1D01E1AD002B40FF /* libReact.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 146834041AC3E56700842450 /* libReact.a */; };
 		146834051AC3E58100842450 /* libReact.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 146834041AC3E56700842450 /* libReact.a */; };
-		1916AD410B0B45C9BEDB553E /* libA0RNLock.a in Frameworks */ = {isa = PBXBuildFile; fileRef = CBED167597174594A47B20BE /* libA0RNLock.a */; };
```

Comment:
It seems that you are removing all Auth0 dependencies, this kind-of breaks Auth0 :)

Here's what I get when configuring it (i.e. it's confirmed to be broken):

```
69Possible Unhandled Promise Rejection (id: 0):
Cannot read property 'registerColorForKey' of undefined
TypeError: Cannot read property 'registerColorForKey' of undefined
    at http://localhost:8081/index.ios.bundle?platform=ios&dev=true&minify=false:97042:11
    at Array.forEach (native)
    at Auth0Lock.customizeTheme (http://localhost:8081/index.ios.bundle?platform=ios&dev=true&minify=false:97040:22)
    at Object.showLogin (http://localhost:8081/index.ios.bundle?platform=ios&dev=true&minify=false:96935:6)
    at Constructor.componentWillReceiveProps (http://localhost:8081/index.ios.bundle?platform=ios&dev=true&minify=false:96418:7)
    at http://localhost:8081/index.ios.bundle?platform=ios&dev=true&minify=false:27229:13
    at measureLifeCyclePerf (http://localhost:8081/index.ios.bundle?platform=ios&dev=true&minify=false:26695:8)
    at ReactCompositeComponentWrapper.updateComponent (http://localhost:8081/index.ios.bundle?platform=ios&dev=true&minify=false:27228:1)
    at ReactCompositeComponentWrapper.receiveComponent (http://localhost:8081/index.ios.bundle?platform=ios&dev=true&minify=false:27165:6)
    at Object.receiveComponent (http://localhost:8081/index.ios.bundle?platform=ios&dev=true&minify=false:25340:18)
    at ReactCompositeComponentWrapper._updateRenderedComponent (http://localhost:8081/index.ios.bundle?platform=ios&dev=true&minify=false:27372:17)
    at ReactCompositeComponentWrapper._performComponentUpdate (http://localhost:8081/index.ios.bundle?platform=ios&dev=true&minify=false:27342:6)
    at ReactCompositeComponentWrapper.updateComponent (http://localhost:8081/index.ios.bundle?platform=ios&dev=true&minify=false:27263:6)
    at ReactCompositeComponentWrapper.performUpdateIfNecessary (http://localhost:8081/index.ios.bundle?platform=ios&dev=true&minify=false:27179:6)
    at Object.performUpdateIfNecessary (http://localhost:8081/index.ios.bundle?platform=ios&dev=true&minify=false:25372:18)
    at runBatchedUpdates (http://localhost:8081/index.ios.bundle?platform=ios&dev=true&minify=false:24983:17)
4debugger-ui:1 Error in event handler for (unknown): TypeError: Cannot read property 'shortcutLock' of undefined
    at chrome-extension://oknpjjbmpnndlpmnhmekjpocelpnlfdi/js/content.js:32:32
```